wielder/util/wgit.py

```diff
@@ -37,6 +38,22 @@ def __init__(self, repo_path):
 
             self.branch = branch
 
+    def dev_info(self):
```

Comment:
It seems to me that the developer info is accumulated rather than replaced. 

Please add ne lines for each submodule, the output should be human friendly